packages/quantic/force-app/main/default/lwc/quanticTab/quanticTab.html

```diff
@@ -1,7 +1,15 @@
 <template>
   <template if:true={shouldDisplay}>
     <li class={tabClass} title={label} role="presentation">
```

Comment:
I would move the title of this `li` to the `button`, in case some screen readers try to read it.

Also, if this `li` has a parent `ul`/`ol`, I would not overwrite the role on either of them, I think it makes sense to have a list of tabs. But that's subjective.

Reply:
switched `li`,`ui`/`ol` to `div`